### DISCUSSION TITLE
官方给出的模型大小和自己训练出来的模型大小差距很大

### DISCUSSION THREAD

**Discussion by leidriver201120** [STVIR/pysot#579] [Q&A]
官方给出的SiamRPN++只有206m，但是我使用不一样的数据集，但是不该动训练配置，train得到的pth文件先是244m训练backbone后是412m大小，这个大小跟官方给出的大小差别很大呢？
并且我检查过trainable_params，它在训练backbone之前只有neck层和head层的参数，应该是没有问题的啊。
而且官方给出的预训练ResNet50就已经有178m了，按照官方给出的模型pth大小，其他参数只有28m吗难道？？
没搞懂为什么pth文件大小差异这么大

**Comment by vic01512** [comment]
我的情况也是一样的
